dashboards-observability/public/components/explorer/home.tsx

```diff
@@ -114,10 +114,12 @@ export const Home = (props: IHomeProps) => {
       sortOrder: 'desc',
       fromIndex: 0,
     });
-    const nonAppVisualizations = res.observabilityObjectList.filter(
-      (object: any) => object.savedVisualization && !object.savedVisualization.application_id
+    const nonAppObjects = res.observabilityObjectList.filter(
+      (object: any) =>
```

Comment:
Can we provide a more specific type than `any` here?

Reply:
Done! 😄